### ISSUE TITLE
(Python-)Setup seems to miss something

### ISSUE THREAD

**Issue by Matthies** [kiudee/chess-tuning-tools#104]
* Chess Tuning Tools version: latest stable
* Python version: 3.8.4
* Operating System: Win10

First of all: I have almost no experience with Python and Python setup so please forgive me if problem is too stupid.

What I did:
1. Install Windows x86_64 python install package from https://www.python.org/downloads/release/python-384/
2. Start a commandline and run "python -m pip install chess-tuning-tools"
This seemed to work fine, there was some little warning about newer pip avalable but I guess this is not important.
3. Created a folder C:\Entwicklung\chesstune and put the opening file 2moves_v1.pgn, the cutechess-cli.exe, the engine.exe and a very simple_tune.json into that folder.
4. From a commandline in that folder I ran  the tune command and it gave me
```
C:\Entwicklung\chesstune>tune local -c simple_tune.json
Traceback (most recent call last):
  File "c:\users\andreas\appdata\local\programs\python\python38\lib\runpy.py", line 194, in _run_module_as_main
    return _run_code(code, main_globals, None,
  File "c:\users\andreas\appdata\local\programs\python\python38\lib\runpy.py", line 87, in _run_code
    exec(code, run_globals)
  File "C:\Users\andreas\AppData\Local\Programs\Python\Python38\Scripts\tune.exe\__main__.py", line 4, in <module>
  File "c:\users\andreas\appdata\local\programs\python\python38\lib\site-packages\tune\cli.py", line 13, in <module>
    from bask.optimizer import Optimizer
  File "c:\users\andreas\appdata\local\programs\python\python38\lib\site-packages\bask\__init__.py", line 10, in <module>
    from .optimizer import Optimizer
  File "c:\users\andreas\appdata\local\programs\python\python38\lib\site-packages\bask\optimizer.py", line 4, in <module>
    from arviz import hdi
  File "c:\users\andreas\appdata\local\programs\python\python38\lib\site-packages\arviz\__init__.py", line 28, in <module>
    from .data import *
  File "c:\users\andreas\appdata\local\programs\python\python38\lib\site-packages\arviz\data\__init__.py", line 2, in <module>
    from .inference_data import InferenceData, concat
  File "c:\users\andreas\appdata\local\programs\python\python38\lib\site-packages\arviz\data\inference_data.py", line 15, in <module>
    from ..utils import _subset_list, HtmlTemplate
  File "c:\users\andreas\appdata\local\programs\python\python38\lib\site-packages\arviz\utils.py", line 670, in <module>
    class HtmlTemplate:
  File "c:\users\andreas\appdata\local\programs\python\python38\lib\site-packages\arviz\utils.py", line 696, in HtmlTemplate
    css_template = f"<style> {xr.core.formatting_html.CSS_STYLE}{specific_style} </style>"
AttributeError: module 'xarray.core.formatting_html' has no attribute 'CSS_STYLE'
```
Has probably nothing to do with the content of the config file, the error is the same calling tune local without a config file.
Any idea?

**Comment by kiudee** [comment]
This is an upstream issue of the arviz package:
https://github.com/kiudee/chess-tuning-tools/issues/101
https://github.com/arviz-devs/arviz/issues/1387

The best fix is to downgrade xarray to 0.16.0.

**Comment by Matthies** [comment]
I downgraded xarray and the tuner runs now. Thanks.
